app/src/main/kotlin/com/proxerme/app/helper/MaterialDrawerHelper.kt

```diff
@@ -133,17 +157,32 @@ class MaterialDrawerHelper(context: Activity, toolbar: Toolbar,
 
     private fun buildDrawer(context: Activity, toolbar: Toolbar, accountHeader: AccountHeader,
                             savedInstanceState: Bundle?): Drawer {
-        return DrawerBuilder(context)
+        val stickyDrawerItems = generateStickyDrawerItems()
+        this.stickyItemIds = stickyDrawerItems.mapNotNull { DrawerItem.fromOrNull(it.identifier) }
+                .toTypedArray()
+
+        val drawer = DrawerBuilder(context)
```

Comment:
`drawer` might not be a good name, since this is no drawer yet. Maybe something like `builder`?